imperial_coldfront_plugin/templates/imperial_coldfront_plugin/overrides/project_detail.html

```diff
@@ -23,6 +25,34 @@ <h2 class="text-justify">{{ project.title }}</h2>
   <hr>
 </div>
 
+<!-- Start Credit Balance Display -->
+<div class="card mb-3 bg-light border-primary">
+  <div class="card-body">
+    <div class="row align-items-center">
+      <div class="col-md-8">
+        <h3 class="card-title mb-0">
+          <i class="fas fa-coins" aria-hidden="true"></i> Credit Balance
+        </h3>
+      </div>
+      <div class="col-md-4 text-right">
+        {% get_project_credit_balance project as credit_balance %}
+        <h2 class="mb-0">
+          {% if credit_balance >= 0 %}
+            <span class="badge badge-success badge-lg" style="font-size: 1.5rem; padding: 0.5rem 1rem;">
+              {{ credit_balance|intcomma }} credits
+            </span>
+          {% else %}
+            <span class="badge badge-danger badge-lg" style="font-size: 1.5rem; padding: 0.5rem 1rem;">
+              {{ credit_balance|intcomma }} credits
+            </span>
```

Comment:
Don't need to duplicate so much here as the only thing changing between the two is the badge class.